src/explore-education-statistics-admin/src/pages/release/data/components/ImporterStatus.tsx

```diff
@@ -161,17 +161,28 @@ const ImporterStatus = ({
         />
       )}
 
-      {currentStatus.errors && currentStatus.errors.length > 0 && (
-        <Details
-          className="govuk-!-margin-top-1 govuk-!-margin-bottom-0"
-          summary="See errors"
-        >
-          <ul className="govuk-!-margin-top-0">
-            {currentStatus.errors.map((error, index) => (
-              <li key={index.toString()}>{error}</li>
-            ))}
-          </ul>
-        </Details>
+      {currentStatus.status === 'FAILED' && (
+        <>
+          {currentStatus.errors && currentStatus.errors.length > 0 && (
```

Comment:
We could use optional chaining here to be more succinct: 
```
currentStatus?.errors?.length > 0 &&
```

Reply:
Unfortunately, you can't actually do this as the compiler won't let you compare undefined with 0 :smile:

Reply:
ah that's true, good catch haha